### PR TITLE
Wrote a blog about Tent - a development only dependency manager powered by Podman

### PR DESCRIPTION
**Pull request by fhsinchy**
I've developed a CLI tool called [Tent](https://github.com/fhsinchy/tent) that allows you to spin up pre-configured containers of development dependencies like `mysql` or `redis` with simple one-liners. I've added a tutorial on how to install and use this program.

### CODE REVIEW

File: _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+title: Easy Development Dependency Management With Podman and Tent
+layout: default
+author: <author id, from the example above 'jsmith'>
```
Comment (TomSweeneyRedHat) [inline _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md]:
```suggestion
author: fhsinchy
```

File: _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+categories: [blogs]
+tags: <your tags here>
```
Comment (TomSweeneyRedHat) [inline _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md]:
```suggestion
tags: tent, hpc, podman, containers
```
and more as you see fit?

File: _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+| CONTAINER              | Image               | PORTS          |
++--------------+----------------+---------------+---------------+
+| tent-mysql-5.7-3307    | docker.io/mysql:5.7 | 3307->3306/tcp |
+| tent-mysql-latest-3306 | docker.io/mysql:5.7 | 3306->3306/tcp |
++--------------+----------------+---------------+---------------+
+```
+
+## Container Management
+
+Containers started by `tent` are regular containers with some pre-set configurations. So you can use regular `podman` commands such as `ls`, `inspect`, `logs` etc on them. Although `tent` comes with a `list` command, using the `podman` commands will result in more informative results. The target of `tent` is to provide plug and play containers, not to become a full-fledged `podman` cli.
```
Comment (TomSweeneyRedHat) [inline _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md]: Can you add a "new" page too, similar to https://github.com/containers/podman.io/blob/master/_posts/2021-01-23-new.md?  This page here will be listed on the "Blog" page.  The "new" page will be shown on the main page and should just be a brief description with a link to the page.

File: _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+![podman logo](https://podman.io/images/podman.svg)
+
+{% assign author = site.authors[page.author] %}
+# Easy Development Dependency Management With Podman and Tent
+## By {{ author.display_name }} [GitHub](https://github.com/{{ author.github }}) [Twitter](https://twitter.com/{{ author.twitter }})
+
+Installing and managing development dependencies for various project is a chore and one thing that can improve your everyday workflow is the usage of containers.
+
+[Tent](https://github.com/fhsinchy/tent/) is a CLI tool for running development dependencies such as MySQL, Mongo, ElasticSearch etc inside pre-configured containers using simple one-liners.
+
```
Comment (TomSweeneyRedHat) [inline _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md:17]: Please add a readmore tag here or after the first paragraph. 

`<!--readmore-->`

**Review by TomSweeneyRedHat — COMMENTED** [review]
Neat tool, I was not aware of this and it will be an excellent blog post.  Just a few changes to make the jekyll system a bit happier, but the main post looks good.
Thanks!

**Comment by fhsinchy** [comment]
I'll definitely update this by tomorrow. Thanks a lot for your compliments.

**Comment by fhsinchy** [comment]
@TomSweeneyRedHat First of all my apologies for not adding the meta and the `readmore` tag properly, I'll be much more cautious in the future. I've made the suggested changes. Please let me know if they've been done properly or not.

**Comment by TomSweeneyRedHat** [comment]
LGTM
THanks @fhsinchy !
@rhatdan PTAL and merge if it LGTY

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**